Dockerfile

```diff
@@ -1217,11 +1217,12 @@ COPY --from=sdk-plus-lz4 /usr/share/licenses/lz4 /usr/share/licenses/lz4
 # a format that's convenient for extracting later.
 COPY --from=toolchain-archive /toolchain /toolchain
 
-# "sdk-libc-musl" has the musl C library and headers. We omit "sdk-libc-gnu"
-# because we expect to build glibc again for the target OS, while we will use
-# the musl artifacts directly to generate static binaries such as migrations.
+# "sdk-libc-musl" has the MUSL C library and headers.
 COPY --from=sdk-libc-musl / /
 
+# "sdk-libc-gnu" has the GNU C library and headers.
+COPY --from=sdk-libc-gnu / /
```

Comment:
Why is this showing up as a change here? It was merged in https://github.com/bottlerocket-os/bottlerocket-sdk/commit/8ef6f479f.

Reply:
My best guess is that either I (more likely) or git (less likely) got bewildered in the rebase process. I have rebased on develop to pick up the base image change, and checked that this is not showing up as a change in the commit.